README.rst

```diff
@@ -137,184 +189,214 @@ One can also build the plugin using CMake:
+    CMAKE_ARGS="-DENABLE_MPI=ON"  PL_BACKEND="lightning_gpu" python -m pip install -e .
 
-.. code-block:: console
 
-   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" PL_BACKEND="lightning_kokkos" python -m pip install -e .
+Test L-GPU with MPI
+===================
 
-Alternatively, you can install the Python interface with:
+You may test the Python layer of the MPI enabled plugin as follows:
 
 .. code-block:: console
 
-   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" PL_BACKEND="lightning_kokkos" python setup.py build_ext
-   python setup.py bdist_wheel
-   pip install ./dist/PennyLane*.whl --force-reinstall
+    mpirun -np 2 python -m pytest mpitests --tb=short
 
-To build the plugin directly with CMake:
+The C++ code is tested with
 
 .. code-block:: console
 
-   cmake -B build -DKokkos_ENABLE_OPENMP=ON -DPLKOKKOS_BUILD_TESTS=ON -DPL_BACKEND=lightning_kokkos -G Ninja
-   cmake --build build
+    rm -rf ./BuildTests
+    cmake . -BBuildTests -DBUILD_TESTS=1 -DBUILD_TESTS=1 -DENABLE_MPI=ON -DCUQUANTUM_SDK=<path to sdk>
+    cmake --build ./BuildTests --verbose
+    cd ./BuildTests
+    for file in *runner_mpi ; do mpirun -np 2 ./BuildTests/$file ; done;
 
-The supported backend options are "SERIAL", "OPENMP", "THREADS", "HIP" and "CUDA" and the corresponding build options are ``-DKokkos_ENABLE_XXX=ON``, where ``XXX`` needs be replaced by the backend name, for instance ``OPENMP``.
-One can activate simultaneously one serial, one parallel CPU host (e.g. "OPENMP", "THREADS") and one parallel GPU device backend (e.g. "HIP", "CUDA"), but not two of any category at the same time.
-For "HIP" and "CUDA", the appropriate software stacks are required to enable compilation and subsequent use.
-Similarly, the CMake option ``-DKokkos_ARCH_{...}=ON`` must also be specified to target a given architecture.
-A list of the architectures is found on the `Kokkos wiki <https://github.com/kokkos/kokkos/wiki/Macros#architectures>`_.
-Note that "THREADS" backend is not recommended since `Kokkos <https://github.com/kokkos/kokkos-core-wiki/blob/17f08a6483937c26e14ec3c93a2aa40e4ce081ce/docs/source/ProgrammingGuide/Initialization.md?plain=1#L67>`_ does not guarantee its safety.
+.. installation_LGPU-end-inclusion-marker-do-not-remove
 
+.. installation_LKokkos-start-inclusion-marker-do-not-remove
 
-Testing
-=======
+.. _lightning-kokkos-installation:
 
-To test with the ROCm stack using a manylinux2014 container we must first mount the repository into the container:
+Lightning-Kokkos installation
+*****************************
 
-.. code-block:: console
+On linux systems, ``lightning.kokkos`` with the OpenMP backend can be installed by providing the optional ``[kokkos]`` tag:
 
-    docker run -v `pwd`:/io -it quay.io/pypa/manylinux2014_x86_64 bash
+.. code-block:: console
 
-Next, within the container, we install the ROCm software stack:
+    $ pip install pennylane-lightning[kokkos]
 
-.. code-block:: console
+Install L-Kokkos from source
```

Comment:
```suggestion
Install Lightning-Kokkos from source
```

Reply:
Before merging this commit, can we check that it doesn't impact the RST/Sphinx references during the docs-build? `rstcheck` and/or `twine check` against a valid source distribution wheel will tell us.

Reply:
Checked, I saw no issue related to those titles with either tool.